tests/mock-data/mock-invalid-input.json

```diff
@@ -0,0 +1,9 @@
+{
+  "command": "get-images",
+  "options": {
+    "path": "D:/zcvbdty",
+    "namePattern": "hash",
+    "orientation": "landscape",
+    "format": "json",
+  }
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected character ('}' (code 125)): was expecting double-quote to start field name](https://app.codacy.com/gh/pass-team/windows-10-lock-screen-image-extractor/pullRequest?prid=5102014)